test/screenshot/infra/commands/travis.sh

```diff
@@ -13,47 +13,53 @@ function exit_if_external_pr() {
   fi
 }
 
+function print_travis_env_vars() {
+  echo
+  env | grep TRAVIS
```

Comment:
This isn't going to output anything that should be hidden, right?

Reply:
I've checked the output many times and have never seen anything sensitive.

That said, I'm still planning to remove this once the Travis config is stable.